sips/minutes/_posts/2016-08-20-sip-20th-september-minutes.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: sip-landing
+title: SIP Meeting Minutes - 20th September 2016
+---
+
+# Minutes
+
+The following agenda was distributed to attendees:
+
+| Topic | Reviewer |
+| --- | --- |
+| [SIP-NN: Scala Meta SIP](https://github.com/scala/scala.github.com/pull/567) | Iulian Dragos and Josh Suereth |
+| [SIP-21: Spores](http://docs.scala-lang.org/sips/pending/spores.html) | Martin Odersky |
+| [SIP-26: Unsigned Integer Data Types](https://github.com/scala/slip/pull/30) | Martin Odersky |
+| [SIP-27: Trailing commas](https://github.com/scala/scala.github.com/pull/533#issuecomment-232959066) | Eugene Burmako |
+
+Jorge Vicente Cantero was the Process Lead and acting secretary of the meeting.
+
+The following proposal was accepted:
+
+* SIP-27: Trailing commas
+
+The following proposal was rejected:
+
+* SIP-26: Unsigned Integer Data Types
+
+The following proposal will have a follow-up evaluation:
+
+* SIP-21: Spores
+
+The following proposals were numbered:
+
+* SIP-28: Inline (part of Scala Meta SIP)
+* SIP-29: Meta (part of Scala Meta SIP)
+
+## Date, Time and Location
+
+The meeting took place at 5:00pm Central European Time / 8:00am Pacific Daylight
+Time on Tuesday, September 20th, 2016 via Google Hangouts.
+
+Minutes were taken by Jorge Vicente Cantero, acting secretary.
+
+## Attendees
+
+Attendees Present:
+
+* Martin Odersky ([@odersky](github.com/odersky)), EPFL
```

Comment:
https://github.com/scala/scala.github.com/commit/747c80ee93837789196fd08ea87c0e250c913beb

That page works because I already fixed the mistake. I also had to fix your other minutes file.

The changes got merged in about 9 hours ago.